### PR TITLE
Change automatic widget updates to use bootgrid when possible

### PR DESCRIPTION
**Pull request by niddey**
Changes automatic updates to use bootgrid if the widget has ajax enabled.
This will stop the widget from forgetting your search/filters and being scrolled to top on every update.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: resources/views/overview/default.blade.php [hunk 1]
```diff
@@ -609,57 +609,57 @@ function widget_settings(data) {
...
+        const $widget_body = $(`#widget_body_${id}`);
+        const $widget_bootgrid = $(`#widget_body_${id} .bootgrid-table`);
+        const settings = $widget_body.parent().data('settings') == 1 ? 1 : 0;
+
+        if (settings === 1) {
+            $(`#widget_body_${id} .bootgrid-table`).bootgrid('destroy');
+            $(`#widget_body_${id} *`).off();
         }
+
         $.ajax({
```
Comment (murrant) [inline resources/views/overview/default.blade.php:622]: Why call the ajax method if you are just going to throw it away and refresh bootgrid?

**Comment by murrant** [comment]
Nice change @niddey

File: resources/views/overview/default.blade.php (hunk 1 above)
Comment (niddey) [reply resources/views/overview/default.blade.php:626]: I was doing it for the title counter. With a quick search no other widget aside from alerts seems to have a counter in the title, so I have now changed the counter to be updated with the bootgrid and skip the other request.

**Review by murrant — APPROVED** [review]
LGTM, now we just need to change the other widgets to ajax :D

**Pull request merged**